src/js/media-poster-image.js

```diff
@@ -0,0 +1,70 @@
+import { defineCustomElement } from './utils/defineCustomElement.js';
+import {
+  Window as window,
+  Document as document,
+} from './utils/server-safe-globals.js';
+import { MediaUIAttributes } from './constants.js';
+
+const template = document.createElement('template');
+
+template.innerHTML = `
+  <style>
+    :host {
+      height: 100%;
+      width: 100%;
+      position: relative;
+    }
+
+    div {
+      position: absolute;
+      top: 0;
+      left: 0;
+      right: 0;
+      bottom: 0;
+      background-position: center;
+      background-size: cover;
+      background-repeat: no-repeat;
+    }
+
+    :host([${MediaUIAttributes.MEDIA_HAS_PLAYED}]) div {
+      display: none;
+    }
+  </style>
+
+  <div part="placeholder"></div>
+  <div part="image"></div>
+`;
+
+const setBackgroundImage = (el, image) => {
+  el.style['background-image'] = `url('${image}')`;
+}
+
+class MediaPosterImage extends window.HTMLElement {
+  static get observedAttributes() {
+    return [MediaUIAttributes.MEDIA_HAS_PLAYED, 'placeholder-src', 'src'];
```

Comment:
`<image>` has a `lowsrc` attribute already. Any reason we wouldn't just reuse that terminology?

Reply:
Hmmm I like that philosophically, but do we care that it's deprecated?

Reply:
Maybe, if there's a negative understanding of it. I don't really understand why they removed it when there's not a clear alternative. Apparently they suggest making a progressive jpeg... I don't see a way `<picture>` solves the same issue.

I think I still like `lowsrc` but I'm good either way. Up to you.

Reply:
Is `lowsrc` this https://html.com/attributes/img-lowsrc/? If so, it seems to be one of the features that  `<picture>` targets: a way to provide image alternatives depending on the page size or device pixel density.

Neither provide the ability to display one of the sources until the "main" source has finished loading. I think that `placeholder` is more direct and clear as to what it's use is.